apis/core/v1beta1/openstackversion_types.go

```diff
@@ -179,6 +185,9 @@ type OpenStackVersionStatus struct {
 	// where we keep track of the container images for previous versions
 	ContainerImageVersionDefaults map[string]*ContainerDefaults `json:"containerImageVersionDefaults,omitempty"`
 
+	// ServiceDefaults - struct that contains defaults for OSP services that can change over time
+	ServiceDefaults ServiceDefaults `json:"serviceDefaults,omitempty"`
```

Comment:
I need to fix this so it works like ContainerImageVersionDefaults above which is a map[string]. This should effectively make it default to 'false' for older OpenStackVersion instances